src/Button/__tests__/BorderlessButton.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'test-utils';
+import { spy } from 'sinon';
+import assert from 'assert';
+import BorderlessButton from '../BorderlessButton';
+
+describe('Borderless Button', function () {
+  let getByText;
+  let clickSpy;
+  beforeEach(function () {
+    clickSpy = spy();
+    ({ getByText } = render(
+      <BorderlessButton clickHandler={clickSpy}>
+        Click Me
+      </BorderlessButton>
+    ));
+  });
+  afterEach(function () {
+    clickSpy.resetHistory();
+    cleanup();
+  });
+  it('renders', function () {
+    getByText('Click Me');
+  });
+  it('calls the click handler when clicked', function () {
+    fireEvent.click(getByText('Click Me'));
+    assert.equal(clickSpy.callCount, 1);
```

Comment:
We should (almost) always use `assert.strictEqual` instead of `assert.equal`, since it compares both the value and type. This is not a major risk here, but if `click.callCount` return `true`, it would still pass, even though we need to see the value be `1`;

_This applies to other button tests as well._

Reply:
Fixed through d6fad8a